corehq/apps/app_manager/models.py

```diff
@@ -1124,9 +1124,9 @@ def _get_media_by_language(self, media_attr, lang, strict=False):
         media_dict = getattr(self, media_attr)
         if not media_dict:
             return None
-        if lang in media_dict:
+        if media_dict.get(lang, ''):
             return media_dict[lang]
```

Comment:
Sorry. Should have read the commit message. This should be:

    if lang in media_dict and media_dict[lang]:
        return media_dict[lang]